warthog_description/urdf/warthog.urdf.xacro

```diff
@@ -351,4 +351,8 @@
 
   <!-- Optional custom includes. -->
   <xacro:include filename="$(optenv WARTHOG_URDF_EXTRAS empty.urdf)" />
+
+  <!-- Optional for Clearpath internal softwares -->
+  <xacro:include filename="$(optenv CPR_URDF_EXTRAS cpr_extras.urdf)" />
```

Comment:
How about just reusing the existing empty URDF?

Reply:
I thought about that. The only problem with that is if someone add something to the empty.urdf ...